modules/swagger-codegen/src/main/resources/JavaSpring/libraries/spring-boot/pom.mustache

```diff
@@ -44,6 +44,10 @@
             <artifactId>spring-boot-starter-tomcat</artifactId>
             <scope>provided</scope>
         </dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
```

Comment:
Will you at least have access to your gh account ?
In that case I can PR to your fork and you will just have to merge.